### ISSUE TITLE
FormData error

### ISSUE THREAD

**Issue by prk-Jr** [flutterchina/dio#1184]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info
Can not upload multiple files via FormData in Dio 4.0.0 but easily uploadable in 3.0.9  

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  |  flutter                     |      |
| Platform Version               | 2.2.1                           |      |
| Dio Version                                              |  4.0.0     |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.
